### ISSUE TITLE
Both Skull Types Placed = Memory Leak?

### ISSUE THREAD

**Issue by Mnesikos** [Alex-the-666/Ice_and_Fire#797]
Having at least one fire dragon skull and one ice dragon skull placed down (near each other, distance wasn't tested) causes memory usage to steadily increase, possibly infinitely. Tested with just Ice and Fire and LLibrary, in both single player and a server.

**Comment by mser7** [comment]
It happen to single skull as well

**Comment by Mnesikos** [comment]
I did various tests with another person, and one skull did not cause us any issue memory-wise. We removed our two skulls from our server that had caused us to spike up to 90% memory usage over a few hours, and that completely halted the memory usage issue. We placed one back down, which had no effect on memory, we removed it and placed the other kind down, still no effect on memory. The continuous memory usage increase only occurred when BOTH a fire and ice skull are placed in the world near each other. At least that has been what I have seen.
